src/main/kotlin/org/quiltmc/community/App.kt

```diff
@@ -124,6 +125,13 @@ suspend fun setupQuilt() = ExtensibleBot(DISCORD_TOKEN) {
             commandCheck { hasPermission(Permission.Administrator) }
         }
 
+        moderation {
+            loggingChannelName = "cozy-logs"
+
+            commandCheck { hasBaseModeratorRole() }
+            commandCheck { inQuiltGuild() }
```

Comment:
This is nitpicky, but put the guild check first - it's a less API-intensive operation in most situations

Reply:
Done in de098e3